howto-gallery/scripts/Equipment.ts

```diff
@@ -30,21 +28,23 @@ export function givePlayerEquipment(log: (message: string, status?: number) => v
   const armorStandLoc = { x: targetLocation.x, y: targetLocation.y, z: targetLocation.z + 4 };
   let armorStand = players[0].dimension.spawnEntity("armor_stand", armorStandLoc);
 
-  const equipmentCompPlayer = players[0].getComponent("equippable") as mc.EntityEquippableComponent;
-  equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Head, new mc.ItemStack("minecraft:golden_helmet"));
-  equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Chest, new mc.ItemStack("minecraft:iron_chestplate"));
-  equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Legs, new mc.ItemStack("minecraft:diamond_leggings"));
-  equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Feet, new mc.ItemStack("minecraft:netherite_boots"));
-  equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Mainhand, new mc.ItemStack("minecraft:wooden_sword"));
-  equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Offhand, new mc.ItemStack("minecraft:shield"));
-
-  const equipmentCompArmorStand = armorStand.getComponent(
-    "equippable"
-  ) as mc.EntityEquippableComponent;
-  equipmentCompArmorStand.setEquipment(mc.EquipmentSlot.Head, new mc.ItemStack("minecraft:golden_helmet"));
-  equipmentCompArmorStand.setEquipment(mc.EquipmentSlot.Chest, new mc.ItemStack("minecraft:iron_chestplate"));
-  equipmentCompArmorStand.setEquipment(mc.EquipmentSlot.Legs, new mc.ItemStack("minecraft:diamond_leggings"));
-  equipmentCompArmorStand.setEquipment(mc.EquipmentSlot.Feet, new mc.ItemStack("minecraft:netherite_boots"));
-  equipmentCompArmorStand.setEquipment(mc.EquipmentSlot.Mainhand, new mc.ItemStack("minecraft:wooden_sword"));
-  equipmentCompArmorStand.setEquipment(mc.EquipmentSlot.Offhand, new mc.ItemStack("minecraft:shield"));
+  const equipmentCompPlayer = players[0].getComponent("equippable");
+  if (equipmentCompPlayer) {
+    equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Head, new mc.ItemStack("minecraft:golden_helmet"));
+    equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Chest, new mc.ItemStack("minecraft:iron_chestplate"));
+    equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Legs, new mc.ItemStack("minecraft:diamond_leggings"));
+    equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Feet, new mc.ItemStack("minecraft:netherite_boots"));
+    equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Mainhand, new mc.ItemStack("minecraft:wooden_sword"));
+    equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Offhand, new mc.ItemStack("minecraft:shield"));
```

Comment:
Can you use `MinecraftItemTypes` for these?

Reply:
would be great to have a linting rule for this XD

Reply:
Done

Reply:
Huh... that's an interesting idea? Maybe we track a brainstorming item for this? It's tricky though because its kind of a suggestion, and it would be valid to pass a non minecraft string in here. So maybe if the parameter is a string starting with "minecraft:" 🤔 ?